src/components/schemas/expense.yaml

```diff
@@ -1678,29 +1678,81 @@ expense_delete_in:
 
 
 expenses_stats_in:
+  type: object
+  required:
+    - data
   properties:
     query_params:
-      nullable: False
+      nullable: false
```

Comment:
Is this case insensitive? If not, then was it wrong before? Can you please check this?

Reply:
[Swagger docs](https://swagger.io/docs/specification/v3_0/data-models/data-types/#null) use lowercase all over the place. Both seem to work though.